README.md

```diff
@@ -515,10 +515,21 @@ Options:
   Default is `false.`
 * `dangerousExtend`: Disable security checks for `extend` query.
   Default is `false.`
-* `mobileToDesktop`: Use desktop browsers if Can I Use doesn’t have data
-  about this mobile version. For instance, Browserslist will return
-  `chrome 20` on `and_chr 20` query (Can I Use has only data only about
-  latest versions of mobile browsers). Default is `false`.
+* `mobileToDesktop`: Resolve the mobile version using the desktop version when
+  Can I Use doesn't have data about the specified version. For instance, it
+  will return two Chrome for Android versions even if Can I Use has only
+  data only about the latest version. Default is `false`.
+* `normalizers`: An array of names of normalizers below and functions
+  that accept a browser and return the normalized versions, which are
+  applied to resolved browsers in order. For instance, `['toChromium', 'toDesktop']`.
+  * `toChromium`: Normalize Chromium-based browsers to Chrome. For instance,
```

Comment:
The name may be misleading as it also normalizes Gecko-based browsers to Firefox, but I cannot think of better name.

Reply:
We can use `byEngine`